packages/permissions/src/components/authorized/authorized.tsx

```diff
@@ -25,13 +28,38 @@ type TActionRight = {
 type TActionRights = {
   [key: string]: TActionRight;
 };
+// Data fences
+type TDataFenceGroupedByPermission = {
+  // E.g. { canManageOrders: { values: [] } }
+  [key: string]: { values: string[] } | null;
```

Comment:
@emmenko do you have any idea why is values an array of strings and not of booleans ? I'm confused of what should be inside now 🤔

Reply:
Maybe the storeKey?

Reply:
It should be the store keys as far as I understood from Adnan...

Reply:
of course! thank you :D

Reply:
Yes it should be storekeys